File: crates/deslop/tests/fixtures/fused-golden-typescript/rename_lean_b.ts
export function assessParcelLevy(parcels: number[], levyShare: number): number {
  let weightTotal = 0;
  for (const parcelMass of parcels) {
    weightTotal = weightTotal + parcelMass;
  }
  const levyAmount = weightTotal * levyShare;
  const weightBurden = weightTotal + levyAmount;
  return weightBurden;
}

File: crates/deslop/tests/fixtures/fused-golden-python/rename_lean_b.py
def assess_parcel_levy(parcels, levy_share):
    weight_total = 0
    for parcel_mass in parcels:
        weight_total = weight_total + parcel_mass
    levy_amount = weight_total * levy_share
    weight_burden = weight_total + levy_amount
    return weight_burden
